src/component/card/dataverseCard/dataverseCard.tsx

```diff
@@ -36,6 +39,10 @@ ${description}`,
     []
   )
 
+  const handleDataverseItemDetails = (id: string) => () => {
+    navigate(`/dataverse/${type}/${id}`)
+  }
```

Comment:
Unexpected closure here. You need to make a decision between two options: passing all values as arguments (id, type) or capturing all values.

Reply:
I missed it, thanks